pkg/kubelet/cm/devicemanager/device_plugin_stub.go

```diff
@@ -123,6 +123,9 @@ func (m *Stub) Start() error {
 		defer m.wg.Done()
 		m.server.Serve(sock)
```

Comment:
m.server.Serve(sock) will block, so other forms of synchronization would be tricky.